spec/paperclip/thumbnail_spec.rb

```diff
@@ -211,10 +211,10 @@
         end
 
         it "errors when trying to create the thumbnail" do
-          assert_raises(Paperclip::Error) do
-            silence_stream(STDERR) do
+          silence_stream(STDERR) do
+            expect {
```

Comment:
Hmm. A continual conundrum for me.

```ruby
expect do
  blah
end.to raise_error
```

I think I prefer `{...}`, but what's your take?